tests/fixtures/hub.py

```diff
@@ -15,6 +17,29 @@
 ENDPOINT_STAGING = "https://hub-ci.huggingface.co"
 ENDPOINT_STAGING_DATASETS_URL = ENDPOINT_STAGING + "/datasets/{repo_id}/resolve/{revision}/{path}"
 
+STAGING_TOKEN_PATH = os.path.expanduser("~/.huggingface/staging_token")
```

Comment:
You can rename it HUB_CI_TOKEN_PATH (we recently changed from moon-staging.huggingface.co to hub-ci.huggingface.co)

same for ENDPOINT_STAGING  and ENDPOINT_STAGING_DATASETS_URL